packages/core/src/render3/error_code.ts

```diff
@@ -38,8 +38,25 @@ export class RuntimeError extends Error {
   }
 }
 
+// Contains a set of error messages that have details guides at angular.io.
+// Full list of available error guides can be found at https://angular.io/errors
+export const HAS_GUIDE = new Set([
+  RuntimeErrorCode.EXPRESSION_CHANGED_AFTER_CHECKED, RuntimeErrorCode.CYCLIC_DI_DEPENDENCY,
+  RuntimeErrorCode.PROVIDER_NOT_FOUND, RuntimeErrorCode.MULTIPLE_COMPONENTS_MATCH,
+  RuntimeErrorCode.EXPORT_NOT_FOUND
+]);
+
 /** Called to format a runtime error */
 export function formatRuntimeError(code: RuntimeErrorCode, message: string): string {
   const fullCode = code ? `NG0${code}: ` : '';
-  return `${fullCode}${message}`;
+
+  let errorMessage = `${fullCode}${message}`;
+  if (HAS_GUIDE.has(code)) {
```

Comment:
I'd like to propose a slightly different name for the `HAS_GUIDE` const so that this condition looks a bit nicer :)
```suggestion
  if (ERRORS_WITH_GUIDES.has(code)) {
```